### PR TITLE
workflow-launch: Add Windows node config for GCP

### PR DESCRIPTION
**Pull request by jfrancoa**
This commit adds a ci-chat-bot workflow config for launching an IPI cluster in GCP with a parametrized number of Windows workers and runs the common end-to-end test suite.

### CODE REVIEW

**Comment by jfrancoa** [comment]
rehearsal testing this workflow can be found in https://github.com/openshift/release/pull/37431

**Comment by jfrancoa** [comment]
ping @saifshaikh48 @sebsoto , could you please review this patch before asking for approval? thanks!

File: ci-operator/step-registry/openshift/e2e/gcp/windows-node/openshift-e2e-gcp-windows-node-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    pre:
+      - chain: ipi-conf-gcp
+      - ref: ovn-conf
+      - ref: ovn-conf-hybrid-manifest
+      - chain: ipi-install
+      - ref: operatorhub-subscribe
+      - ref: windows-conf-operator
+      - ref: ipi-conf-gcp-windows-machineset
+    test:
+      - ref: openshift-e2e-test
```
Comment (saifshaikh48) [inline ci-operator/step-registry/openshift/e2e/gcp/windows-node/openshift-e2e-gcp-windows-node-workflow.yaml:18]: This is the [general OCP payload test suite](https://steps.ci.openshift.org/reference/openshift-e2e-test), not related to Windows
The WinC test step is named [windows-e2e-operator-test](https://steps.ci.openshift.org/reference/windows-e2e-operator-test)

**Comment by jfrancoa** [comment]
> /lgtm
> 
> I'm curious what the purpose is for invoking the general openshift e2e tests though

I used the same value from the [initial workflow in AWS ](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/aws/windows-node/openshift-e2e-aws-windows-node-workflow.yaml#L18), I guess, as this workflow was intended to be used from cluster-bot it didn't pay much attention to the test ref, cluster-bot runs pre to provision and post to deprovision...

**Review by danilo-gemoli — APPROVED** [review]

**Pull request merged**